### PR TITLE
Use trusty distribution for testing

### PR DESCRIPTION
**Pull request by vojtasvoboda**
Make Lumen build green again.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks, but this actually results in older versions of php getting pulled in. We only need to activate this on the actual HHVM build.

File: .travis.yml [hunk 1]
```diff
@@ -2,12 +2,14 @@ language: php
 
 php:
   - 5.6
-  - 7.0
+  - 7.0.19
```
Comment (GrahamCampbell) [inline .travis.yml]: Please don't do this. Also, php 5.6 is loaded at an older version too. Just only do dist trusty for hhvm.

**Comment by GrahamCampbell** [comment]
```
matrix:
  include:
    - php: 5.6
    - php: 7.0
    - php: 7.1
    - php: hhvm
      dist: trusty
```

File: .travis.yml (hunk 1 above)
Comment (vojtasvoboda) [inline .travis.yml]: Yes, that was a bad idea, I just wanted to try if we can force latest PHP version. CircleCI config is updated now.

**Comment by GrahamCampbell** [comment]
> Use CircleCI build matrix

We don't use CircleCI, but meh. :P

**Comment by vojtasvoboda** [comment]
Sorry for typo :-) Now it's ready for merge. Thanks for the help!

**Comment by taylorotwell** [comment]
Why not just make it green without requring trusty?

**Comment by GrahamCampbell** [comment]
> Why not just make it green without requring trusty?

Because Facebook have dropped support for HHVM on that OS. It's not that our tests fail, it's that they don't run.

**Comment by taylorotwell** [comment]
We shouldn't be running tests on HHVM at all I don't think? We don't on the main Laravel repository?

**Pull request closed without merging**